### ISSUE TITLE
Can't boot cluster with VMware Fusion 7

### ISSUE THREAD

**Issue by jimmycuadra** [coreos/coreos-vagrant#204]
With a fresh checkout of the repository, commit 73c8d2, against Vagrant 1.7.2, vagrant-vmware-fusion 3.2.0, And VMware Fusion 7.1.0:

```
$ vagrant up
Bringing machine 'core-01' up with 'vmware_fusion' provider...
Bringing machine 'core-02' up with 'vmware_fusion' provider...
Bringing machine 'core-03' up with 'vmware_fusion' provider...
Bringing machine 'core-04' up with 'vmware_fusion' provider...
Bringing machine 'core-05' up with 'vmware_fusion' provider...
==> core-01: Checking if box 'coreos-alpha' is up to date...
==> core-01: Verifying vmnet devices are healthy...
==> core-01: Preparing network adapters...
==> core-01: Starting the VMware VM...
==> core-01: Waiting for machine to boot. This may take a few minutes...
    core-01: SSH address: 192.168.124.130:22
    core-01: SSH username: core
    core-01: SSH auth method: private key
==> core-01: Machine booted and ready!
==> core-01: Forwarding ports...
    core-01: -- 22 => 2222
==> core-01: Setting hostname...
==> core-01: Configuring network adapters within the VM...
==> core-02: Cloning VMware VM: 'coreos-alpha'. This can take some time...
==> core-02: Checking if box 'coreos-alpha' is up to date...
==> core-02: Verifying vmnet devices are healthy...
==> core-02: Preparing network adapters...
==> core-02: Fixed port collision for 22 => 2222. Now on port 2200.
==> core-02: Starting the VMware VM...
==> core-02: Waiting for machine to boot. This may take a few minutes...
    core-02: SSH address: 192.168.124.131:22
    core-02: SSH username: core
    core-02: SSH auth method: private key
==> core-02: Machine booted and ready!
==> core-02: Forwarding ports...
    core-02: -- 22 => 2200
==> core-02: Setting hostname...
==> core-02: Configuring network adapters within the VM...
==> core-03: Cloning VMware VM: 'coreos-alpha'. This can take some time...
==> core-03: Checking if box 'coreos-alpha' is up to date...
==> core-03: Verifying vmnet devices are healthy...
==> core-03: Preparing network adapters...
==> core-03: Fixed port collision for 22 => 2222. Now on port 2201.
==> core-03: Starting the VMware VM...
An error occurred while executing `vmrun`, a utility for controlling
VMware machines. The command and output are below:

Command: ["start", "/Users/jimmy/Code/coreos-vagrant/.vagrant/machines/core-03/vmware_fusion/d7cbbad7-c6bd-4486-ac28-c4954a5273ff/coreos_production_vagrant_vmware_fusion.vmx", "nogui", {:notify=>[:stdout, :stderr]}]

Stdout: 2015-01-30T17:45:14.064| ServiceImpl_Opener: PID 1564
Error: The operation was canceled

Stderr:
```

I've also tried with GUI mode turned on but it just hangs on the second or third box. A single box will boot successfully, but not a cluster. VirtualBox works as expected.

**Comment by gorzell** [comment]
I was having a similar issue.  I think it might be related to how much free memory there is on your machine, or having virtualbox running at the same time.  The error message is certainly not helpful.

Basically I noticed that I had another VM running and once I stopped it and then tried running coreos again it worked fine.

Unfortunately I haven't been able to repro to see if more detailed logs have a better error message.
